docs/guide/installation.md

```diff
@@ -76,12 +76,12 @@ poetry install --only main
 # Install nix. If you have installed via another manager, remove and use this install (from https://nixos.org/download)
 sh <(curl -L https://nixos.org/nix/install) --daemon
 
-# Enable nix-command and flakes experimental features for nix:
+# Enable nix-command and flakes experimental features for nix (you may need to nano into /etc/nix/nix.conf and add 'experimental-features = nix-command flakes' manually):
 echo 'experimental-features = nix-command flakes' >> /etc/nix/nix.conf
 
-# Add cachix for cached binaries
-nix-env -iA cachix -f https://cachix.org/api/v1/install
-cachix use lnbits
+# Add cachix for cached binaries (you may need to install cachix `sudo apt install nix-bin`)
```

Comment:
no need to install nix-bin

* cachix is installed via the `nix-env` below
* nix-env should be installed via the `sh <(curl -L https://nixos.org/nix/install)` above